translations/base-de.yaml

```diff
@@ -193,64 +193,59 @@ dialogs:
     editConstantProducer:
         title: Set Item
     puzzleLoadFailed:
-        title: Puzzles failed to load
-        desc: "Unfortunately the puzzles could not be loaded:"
+        title: Puzzle konnte nicht geladen werden.
+        desc: "Leider konnte das Puzzle nicht geladen werden."
     submitPuzzle:
-        title: Submit Puzzle
-        descName: "Give your puzzle a name:"
-        descIcon: "Please enter a unique short key, which will be shown as the icon of
-            your puzzle (You can generate them <link>here</link>, or choose one
-            of the randomly suggested shapes below):"
-        placeholderName: Puzzle Title
+        title: Puzzle einreichen
+        descName: "Gib deinem Puzzle einen Namen:"
+        descIcon: "Bitte gib einen kurzen Code ein, welcher als Icon für dein Puzzle benutzt wird (Du kannst diese <link>hier</link> erstellen, oder suche dir eines der zufällig generierten Icons aus.):"
```

Comment:
Bitte gib hier den Kurz-Code einer Form an. Dieser wird als Icon für das Puzzle verwendet

Punkt in der Klammer muss weg